terraform/cos/variables.tf

```diff
@@ -48,6 +48,12 @@ variable "anti_affinity" {
   default     = true
 }
 
+variable "telemetry_correlation" {
+  description = "Configure Loki, Tempo, and Mimir data sources to enable telemetry correlation across logs, metrics, and traces."
+  type        = bool
+  default     = false
```

Comment:
I don't recall if this was previously discussed, but... why is it not defaulting to True? Is there a downside to enabling it other than the extra size of the storage for exemplars?

Reply:
let's hear @sed-i on this then, but I think this is a powerful feature with low overhead that, if optional at all, should be opt-out rather than opt-in.